src/nanotron/trainer.py

```diff
@@ -198,7 +199,7 @@ def __init__(
 
         # Setup tensorboard write and log writers on output rank
         self.logger_ranks = self.parallel_context.world_rank_matrix[
-            self.unwrapped_model.output_pp_rank, 0, 0
+            0, self.unwrapped_model.output_pp_rank, 0, 0
```

Comment:
would be nice if we do self.parallel_context.world_rank_matrix(expert_rank=0, pp_rank=0, tp_rank=0, dp_rank=0)

Reply:
i very much like this! but that could be in an othogonal PR

Reply:
added a TODO